server/utils/functionalHelpers.js

```diff
@@ -145,3 +147,18 @@ function pickKey(predicate, object) {
 function firstKey(object) {
     return R.keys(object)[0];
 }
+
+function getFieldDetail(fieldPath, fieldConfig) {
+    return R.pipe(
+        R.values,
+        R.head,
+        R.path(fieldPath)
+    )(fieldConfig);
+}
+
+function getFieldName(fieldConfig) {
+    return R.pipe(
+        R.keys,
+        R.head
+    )(fieldConfig);
+}
```

Comment:
Don't know if these should belong here as they are not generic. Maybe in a formConfigMethods file?